.github/workflows/run-tests-nightly.yml

```diff
@@ -0,0 +1,58 @@
+---
+name: Run Test Suite Nightly
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 11 * * *"
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}-${{ github.event_name }}--${{ (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/release/') || startsWith(github.ref, 'refs/heads/long_lived/')) && github.sha || '' }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os.runs-on }}
+    name: All tests
+    timeout-minutes: 30
+    strategy:
+      max-parallel: 4
+      fail-fast: false
+      matrix:
+        os:
+          - runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

Comment:
If fixing the other, might as well update this to @v4 as well (latest version of the action)

Reply:
Done